# An array of points

std_msgs/Int32 trasmittor_id
std_msgs/Int32MultiArray int_array
